Library/Formula/supercollider.rb

```diff
@@ -0,0 +1,61 @@
+
+class Supercollider < Formula
+  homepage "https://supercollider.github.io"
+  url "https://github.com/supercollider/supercollider.git",
+    :tag => "after-qt5",
+    :revision => "085cab288947fe06f468117f5c294db516b2afc0"
+  sha256 "80956465c2637cf0cc67c3550900e2f392d10d33f2518c25bbb9332901ae226c"
+
+  version "3.7-devel"
+
+  depends_on :xcode => :build
+  depends_on :macos => :lion
+  depends_on "cmake" => :build
+  depends_on "qt5" => :build
+  depends_on "readline" => :build
+
+  def install
+    ENV.deparallelize
+    bp = buildpath/"build"
+    mkdir_p bp
+    # std_cmake_args +
+    args = [
+      "-G", "Xcode",
+      "-DCMAKE_PREFIX_PATH=#{Formula["qt5"].opt_prefix}",
+      "-DSC_QT=1",
+      # defaults to the user's OS
+      # "-DCMAKE_OSX_DEPLOYMENT_TARGET=10.7",
+      "-DREADLINE_INCLUDE_DIR=#{Formula["readline"].opt_include}",
+      "-DREADLINE_LIBRARY=#{Formula["readline"].opt_lib}/libreadline.dylib",
+      ".."
+    ]
+
+    cd bp do
+      system "cmake", *args
+      xcodebuild "-target", "install",
+                "-configuration", "Release"
+    end
+    # "SYMROOT=#{bp}" -- fails
+
+    # Copy the app into the cellar
+    # User should then run `brew linkapps supercollider`
+    # to install it into /Applications
+    cp_r "./build/Install/SuperCollider/SuperCollider.app", prefix/"SuperCollider.app"
+
+    # Install sclang and scsynth binaries.
+    # sclang and scsynth need to be executed in the bundle
+    # or the dynamic linking to the Qt libraries gets confused.
+    # write_exec_script creates a wrapper to exec it
+    bin.write_exec_script prefix/"SuperCollider.app/Contents/MacOS/sclang"
+    bin.write_exec_script prefix/"SuperCollider.app/Contents/MacOS/scsynth"
+  end
+
+  test do
+    # `test do` will create, run in and delete a temporary directory.
+    assert File.exist?("#{prefix}/SuperCollider.app")
+    assert File.exist?("#{bin}/sclang")
+    assert File.exist?("#{bin}/scsynth")
```

Comment:
previously I couldn't just run `sclang -V` to check it, but as of yesterday we implemented that so now I can.

otherwise running sclang would start up the language interpreter and there was no easy way to just kill it.